### ISSUE TITLE
'Exit the app' warning

### ISSUE THREAD

**Issue by NripPrajapat** [maelchiotti/LocalMaterialNotes#292]
Add exit warning when I press back button.
It directly exits the app even when I am on the menu list.
It also exits the app directly even when I've selected some notes.

**Comment by maelchiotti** [comment]
Use: https://pub.dev/packages/double_back_to_exit

**Comment by NripPrajapat** [comment]
> Use: https://pub.dev/packages/double_back_to_exit

How do I use it?

You are the developer, so I guess, you would have to implement it.

**Comment by maelchiotti** [comment]
That was a note to myself so I remember which package to use. Sorry for the miscomprehension 😄

**Comment by NripPrajapat** [comment]
When under a category or a label list, if I press back button, then it should go in the main notes list, not just exit the app.

**Comment by maelchiotti** [comment]
I have created a separate issue for this: #293. I'll add what you requested to the list, it already works for the settings and it also makes sens for the labels and the bin.

**Comment by maelchiotti** [comment]
Waiting for #293 to be resolved

**Comment by NripPrajapat** [comment]
It is still not working.

**Comment by maelchiotti** [comment]
Yes, that is simply because I've not managed to do this yet. That's why the issue is still opened, not closed.

I've looked into it a bit and there is a very weird bug that prevents me from intercepting the app closing. I will have to take more time to understand why.
